reconcile/aus/base.py

```diff
@@ -381,163 +499,216 @@ def upgradeable_version(
             version,
             version_data_map,
             ocm.name,
-            policy["workloads"],
-            policy["conditions"],
+            policy.workloads,
+            policy.conditions,
         ):
             return version
     return None
 
 
-def cluster_mutexes(policy: dict) -> list[str]:
+def cluster_mutexes(policy: ConfiguredUpgradePolicy) -> list[str]:
```

Comment:
this convenience access function could be a method in `ConfiguredUpgradePolicy`

Reply:
:heavy_check_mark: